pkg/beacon/mongodb.go

```diff
@@ -151,15 +167,6 @@ func (r *repo) Upsert(uconnMap map[string]*uconn.Input, hostMap map[string]*host
 		writerWorker.start()
 	}
 
-	// grab the local hosts we have seen during the current analysis period
-	// get local hosts only for the summary
-	var localHosts []data.UniqueIP
-	for _, entry := range hostMap {
-		if entry.IsLocal {
-			localHosts = append(localHosts, entry.Host)
-		}
-	}
-
 	// add a progress bar for troubleshooting
 	p = mpb.New(mpb.WithWidth(20))
 	bar = p.AddBar(int64(len(localHosts)),
```

Comment:
This is the line causing the error. If we create a progress bar with length 0, then the library assumes that the progress bar should default to the time of creation (why? who knows...). Since the bar is never filled, the code then locks on the line `p.Wait()` below. 

Please help me double check there aren't any other places where we might initialize a progress bar with length 0. I believe all of the bars which depend on the length of the input maps should be safe since the `fsimporter` ensures that they are filled before calling the analysis package.